packages/records/lib/models/records.ts

```diff
@@ -22,6 +22,17 @@ dayjs.extend(utc);
 
 const BATCH_SIZE = 1000;
 
+export async function getRecordsCount({ connectionId, model }: { connectionId: number; model: string }): Promise<Result<number>> {
```

Comment:
could be moved to a separate file since we are in a models folder (aka one file = one model = one table)
Also missing the environmentId in the where

Reply:
Will leave these methods in this file for now, unless you want to push back. It complicates the namespace, and we end up passing the `recordsService` as a prop some places, and would need to add an additional `recordCountsService` prop there, which to me feels heavy for a simple helper that is pretty tightly coupled to the records service